src/8_logger.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+goog.provide('Logger');
+// jscs:disable validateIndentation
+/**
+ * @constructor
+ */
+
+Logger = function() {
```

Comment:
Since this is a wrapper over the native `console`, should we include a tag or namespace like `BranchSDK`, so that messages emitted from this logger are easily filterable?